tests/unit/test_groupby.py

```diff
@@ -28,7 +29,9 @@ def func(df):
         return df
 
     if parallel:
-        res = scm_run.groupby(*g).apply_parallel(func, n_jobs=-1)
+        res = scm_run.groupby(*g).apply_parallel(
+            func, parallel_processor=get_joblib_parallel_processor(n_jobs=-1)
+        )
```

Comment:
```suggestion
        res = scm_run.groupby(*g).apply_parallel(func)
```

Reply:
My suggested change should do the same thing, I think, and tests also the fallback path in `apply_parallel`.